config/locales/sv.yml

```diff
@@ -591,6 +597,11 @@ sv:
 
     index:
       title:  Hitta H-märkt företag
+      meta:
+        title: "Hitta H-märkt hundföretag, hundinstruktör"
+        description: "Hitta ett etiskt svenskt hundföretag som har tjänat H-märkt. Hitta en hundinstruktör, Hundbolaget, hundens entreprenör."
+        keywords: 'hund, hundägare, hundinstruktör, hundens entreprenör, Hundbolaget, Sveriges Hundföretagare, svenskt hundföretag, etisk, H-markt, ansvarig, tjänat H-marknaden'
```

Comment:
keywords: 'hund, hundägare, hundinstruktör, hundentreprenör, Sveriges Hundföretagare, svenskt hundföretag, etisk, H-märkt, hundkurs'

Reply:
If this entry (key) doesn't exist, then the default will be used.  The default is `meta.default.keywords`  Since these are the same, we can just use the default.

The default are the entries in `meta.defaults` that start on line 8.  If there isn't a `meta....` entry for a page, then one of these defaults will be used instead
https://github.com/AgileVentures/shf-project/blob/c636a2550259172bb8be67793aa2c06a6b580283/config/locales/en.yml#L8